### PR TITLE
CherryPicked master PR 'Filtered CLI, Installer and Longrun tests for upgrade #5182'

### PR DESCRIPTION
**Pull request by jyejare**
$subject

### CODE REVIEW

File: tests/foreman/cli/test_partitiontable.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
 from robottelo.cli.base import CLIReturnCodeError
 from robottelo.cli.factory import make_os, make_partition_table
 from robottelo.cli.partitiontable import PartitionTable
-from robottelo.decorators import run_only_on, skip_if_bug_open, tier1, tier2
+from robottelo.decorators import (
+    run_only_on, skip_if_bug_open, tier1, tier2, upgrade)
```
Comment (san7ket) [inline tests/foreman/cli/test_partitiontable.py]:
```
from robottelo.decorators import (
run_only_on, 
skip_if_bug_open, 
tier1,
tier2,
upgrade
)
```

File: tests/foreman/cli/test_user.py [hunk 2]
```diff
@@ -32,7 +32,8 @@
     valid_emails_list,
     valid_usernames_list,
 )
-from robottelo.decorators import stubbed, skip_if_bug_open, tier1, tier2, tier3
+from robottelo.decorators import (
+    stubbed, skip_if_bug_open, tier1, tier2, tier3, upgrade)
```
Comment (san7ket) [inline tests/foreman/cli/test_user.py]:
```
from robottelo.decorators import (
stubbed, 
skip_if_bug_open, 
tier1, 
tier2, 
tier3, 
upgrade
)
```

**Comment by jyejare** [comment]
@SatelliteQE/quality-engineers , Guys please review and help me to merge this PR asap

File: tests/foreman/cli/test_classparameters.py [hunk 3]
```diff
@@ -43,7 +43,9 @@
     stubbed,
     tier1,
     tier2,
-    bz_bug_is_open)
+    bz_bug_is_open,
```
Comment (renzon) [inline tests/foreman/cli/test_classparameters.py]: This should be alphabetically sorted

**Comment by jyejare** [comment]
@renzon , I fixed your comment, please revisit.

**Pull request merged**